### ISSUE TITLE
Not continuously input the same commands

### ISSUE THREAD

**Issue by livindg** [primefaces/primeng#3424]
**Current behavior**
Input `date` twice, the second result is empty.  

![image](https://user-images.githubusercontent.com/3004916/28310065-d3747e7a-6bdd-11e7-9746-c117da3f6542.png)

**Expected behavior**
Expect to print the result by every command

**environment:**
``` shell
@angular/cli: 1.2.1
node: 8.1.4
os: win32 x64
@angular/animations: 4.3.0
@angular/common: 4.3.0
@angular/compiler: 4.3.0
@angular/core: 4.3.0
@angular/forms: 4.3.0
@angular/http: 4.3.0
@angular/language-service: 4.3.0
@angular/platform-browser: 4.3.0
@angular/platform-browser-dynamic: 4.3.0
@angular/router: 4.3.0
@angular/cli: 1.2.1
@angular/compiler-cli: 4.3.0
```

**Comment by livindg** [comment]
Here is my fixed

![image](https://user-images.githubusercontent.com/3004916/28310214-4ac5d424-6bde-11e7-92e3-15ec63671ff9.png)

**Comment by cagataycivici** [comment]
Fixed via #3733
